packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -69,6 +70,10 @@ const CompleteResetPassword = ({
   };
   const ftlMsgResolver = useFtlMsgResolver();
 
+  useEffect(() => {
+    GleanMetrics.resetPassword.createNewView();
```

Comment:
Should this event only be emitted if the link is valid and the `CompleteResetPassword` page is rendered? (replacing the current `logPageViewEvent(viewName, REACT_ENTRYPOINT);` on L159? Otherwise the ping will be sent out even if the page is not rendered (e.g., if the link is expired or damaged, or if the user has a recovery key and is redirected to the `AccountRecoveryConfirmKey` page).

Reply:
Oh good point.  We do want the event for the render of the password form.  Thanks for catching the error.